aws/logs_monitoring/lambda_function.py

```diff
@@ -48,21 +48,24 @@
 # by default, use the TCP client.
 DD_USE_TCP = os.getenv("DD_USE_TCP", default="false").lower() == "true"
 
+# Set this value to disable SSL over our TCP client.
+# Useful when you are forwarding your logs via TCP to a proxy.
+DD_NO_SSL = os.getenv("DD_NO_SSL", default="false").lower() == "true"
```

Comment:
this lines is very similar to line 49, we could probably create a helper to collect a boolean from environment variables.